### PR TITLE
Style-individual-lab-page

### PR DESCRIPTION
**Pull request by ArshdeepGrover**

### CODE REVIEW

**Comment by ArshdeepGrover** [comment]
Text colour change from #536eff to #42AAFF

**Comment by ArshdeepGrover** [comment]
image

**Comment by ArshdeepGrover** [comment]
image

File: .gitignore [hunk 1]
```diff
@@ -32,6 +32,7 @@ speed-measure-plugin*.json
 
 # Visual Studio Code
 .vscode/*
+.vscode/
```
Comment (arpansac) [inline .gitignore]: fix

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.html [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <ng-template #dialog let-data let-ref="dialogRef">
   <nb-card class="image-popup">
-    <nb-card-header style="align-self: flex-end;">
+    <nb-card-header style="align-self: flex-end">
       <nb-icon (click)="ref.close()" icon="close"></nb-icon>
     </nb-card-header>
 
     <nb-card-body>
-      <img [src]="src" alt="" class="image">
+      <img [src]="src" alt="" class="image" />
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.html]: add the name of the lab - step in this alt

**Review by arpansac — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <ng-template #dialog let-data let-ref="dialogRef">
   <nb-card class="image-popup">
-    <nb-card-header style="align-self: flex-end;">
+    <nb-card-header style="align-self: flex-end">
       <nb-icon (click)="ref.close()" icon="close"></nb-icon>
     </nb-card-header>
 
     <nb-card-body>
-      <img [src]="src" alt="" class="image">
+      <img [src]="src" [alt]="step.name" class="image" />
```
Comment (ArshdeepGrover) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.html:8]: discuss alt name

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab.component.html [hunk 4]
```diff
@@ -4,7 +4,7 @@
       <nb-icon (click)="ref.close()" icon="close"></nb-icon>
     </nb-card-header>
     <nb-card-body>
-      <img [src]="src" alt="Image" class="Image" />
+      <img [src]="src" [alt]="lab.name" class="Image" />
```
Comment (ArshdeepGrover) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab.component.html:7]: check

**Pull request merged**